python/ray/air/util/check_ingest.py

```diff
@@ -133,6 +134,28 @@ def generate_epochs(data: Union[Dataset, DatasetPipeline], epochs: int):
         return train_loop_per_worker
 
 
+def make_local_dataset_iterator(
```

Comment:
OK. I added it originally because I realized there was no public way to create a `DatasetIterator` but it's less of a concern now that we have `Dataset.iterator()`.

Reply:
Hmm so actually I think we should keep something like this because it's actually quite awkward to develop with AIR directly. There's no way to create a local Trainer so to test out the data iteration, etc., you have to rely on print statements or build the right Dataset yourself.

Any objection to keeping this here but renaming to something like `make_dummy_dataset_iterator`?